### ISSUE TITLE
Failed to Init project

### ISSUE THREAD

**Issue by snowdream** [facebook/react-native#5169]
PC: Windows 7
Node:  v4.2.4
Python:  2.7.10 
npm: 2.14.12
Proxy: 127.0.0.1:8087

```bash 
$  react-native init react_native_helloworld --verbose
prompt: Directory react_native_helloworld already exists. Continue?:  (no) yes
This will walk you through creating a new React Native project in D:\workspace\snowdream\git\react_native_helloworld
Installing react-native package from npm...
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at exports._errnoException (util.js:870:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)

```

$ npm config list
; cli configs
user-agent = "npm/2.14.12 node/v4.2.4 win32 x64"

; userconfig C:\Users\yanghui.yangh\.npmrc
nodedir = "D:\\bin\\nodejs"
proxy = "http://127.0.0.1:8087/"
python = "D:\\bin\\Python27\\python.exe"
registry = "http://registry.npmjs.org/"

; builtin config undefined
prefix = "C:\\Users\\yanghui.yangh\\AppData\\Roaming\\npm"

; node bin location = D:\bin\nodejs\node.exe
; cwd = D:\workspace\snowdream\git
; HOME = C:\Users\yanghui.yangh
; 'npm config ls -l' to show all defaults.

**Comment by tdzl2003** [comment]
Hi, I'm looking at this.  This is because of this commit:  [5da0e13](https://github.com/facebook/react-native/commit/5da0e135b6224a2ec9677da3494b8c1e90b06908)

`spawn` cannot run npm on windows.

**Comment by sunnylqm** [comment]
@mkonicek

**Comment by sunnylqm** [comment]
A workaround is to rollback to 0.1.7
```sh
npm i -g react-native-cli@0.1.7
```

**Comment by mkonicek** [comment]
Oh I see, we used to use `exec` before. Thanks for the fix in https://github.com/facebook/react-native/pull/5171 @tdzl2003!

**Comment by mkonicek** [comment]
Just released a new CLI with a fix from https://github.com/facebook/react-native/pull/5171

Can someone using Windows verify?

**Comment by sunnylqm** [comment]
![1](https://cloud.githubusercontent.com/assets/615282/12187240/3276fab6-b5e5-11e5-9162-ad71834ddd27.png)
Verified.

**Comment by snowdream** [comment]
Verified. Fixed #5169
